QUARKUS-4221.md

```diff
@@ -6,10 +6,7 @@ PR: https://github.com/quarkus-qe/quarkus-test-suite/pull/2003
 ## Scope of the testing
 ### What to test
 
-The test coverage will be verified for environments: JVM for Baremetal.
-
- It won't be tested on Native because this support was requested by the Keycloak team, and they explicitly stated (in the comments of the Jira task) that they don't work with native executables,
-so it's not needed at the moment.
+The test coverage will be verified for environments: JVM and Native for Baremetal.
```

Comment:
neither JVM nor Native are environment, I understand what you try to say, but if you don't mind please rephrases